models/MerchandiseItemOptionModel.ts

```diff
@@ -24,19 +24,20 @@ export class MerchandiseItemOptionModel extends BaseEntity {
   @Column('integer', { default: 0 })
   discountPercentage: number;
 
+  // Potential fields: type: string, value: string, position: number
```

Comment:
Potential fields? Isn't this either a NULL value or an object with all three fields?

Reply:
Correct, I added that comment when `metadata` was still of type `object`, but now that it's strongly-ish typed (i.e. has a type but no schema associated with it), the comment is redundant